### PR TITLE
VW: add MY23 Crafter FW

### PR DESCRIPTION
**Pull request by chzollib**
Update Fingerprint with MY23 Crafter, 
The route is here: af1bf21a07172228|2023-06-28--21-00-49--0

### CODE REVIEW

File: selfdrive/car/volkswagen/values.py [hunk 1]
```diff
@@ -392,18 +392,24 @@ def init_make(self, CP: car.CarParams):
...
       b'\xf1\x8704L906056EK\xf1\x896391',
+      b'\xf1\x8705L906023BC\xf1\x892688',
     ],
     # Only current upstreamed vehicle has a manual transmission
     #(Ecu.transmission, 0x7e1, None): [
     #],
     (Ecu.srs, 0x715, None): [
       b'\xf1\x873Q0959655BG\xf1\x890703\xf1\x82\x0e16120016130012051G1313052900',
+      b'\xf1\x875QF959655AG\xf1\x890753\xf1\x82\x13151400151500111111000502----1311120749',
+      b'\xf1\x875QF959655AS\xf1\x890755\xf1\x82\x1315140015150011111100050200--1311120749',
```
Comment (sshane) [inline selfdrive/car/volkswagen/values.py:403]: @jyoung8607 do you know why there might be multiple FW from a single car?

Comment (jyoung8607) [reply selfdrive/car/volkswagen/values.py:403]: There wouldn't be. The PR owner works for a shop that does EV conversions on Crafter family vehicles. He's had openpilot plugged in to at least two of them, and isn't familiar with the process.

I can't see his original route due to age, but `af1bf21a07172228|2023-06-29--14-45-28` seems legitimate for a 2023 MAN TGE (a badge-engineered Crafter).

I've also seen a route where he doesn't have engine firmware; that one was post-conversion. I'll try to support him later on those, but for the time being they won't work with openpilot.

File: selfdrive/car/volkswagen/values.py [hunk 2]
```diff
@@ -392,18 +392,24 @@ def init_make(self, CP: car.CarParams):
...
     ],
     (Ecu.eps, 0x712, None): [
       b'\xf1\x872N0909143E \xf1\x897021\xf1\x82\x05163AZ306A2',
+      b'\xf1\x872N0909144K \xf1\x897045\xf1\x82\x05233AZ810A2',
     ],
     (Ecu.fwdRadar, 0x757, None): [
+      b'\xf1\x872Q0907572AA\xf1\x890396',
       b'\xf1\x872Q0907572M \xf1\x890233',
+      b'\xf1\x872Q0907572R \xf1\x890372',
```
Comment (sshane) [inline selfdrive/car/volkswagen/values.py:412]:
```suggestion
```

File: selfdrive/car/volkswagen/values.py [hunk 3]
```diff
@@ -392,18 +392,24 @@ def init_make(self, CP: car.CarParams):
...
     (Ecu.engine, 0x7e0, None): [
       b'\xf1\x8704L906056EK\xf1\x896391',
+      b'\xf1\x8705L906023BC\xf1\x892688',
     ],
     # Only current upstreamed vehicle has a manual transmission
     #(Ecu.transmission, 0x7e1, None): [
     #],
     (Ecu.srs, 0x715, None): [
       b'\xf1\x873Q0959655BG\xf1\x890703\xf1\x82\x0e16120016130012051G1313052900',
+      b'\xf1\x875QF959655AG\xf1\x890753\xf1\x82\x13151400151500111111000502----1311120749',
```
Comment (sshane) [inline selfdrive/car/volkswagen/values.py:402]:
```suggestion
```

**Comment by sshane** [comment]
@chzollib if you have a route for the other FW, we can add that in a new PR

**Pull request merged**